dastardcommander/observe.py

```diff
@@ -284,6 +285,22 @@ def handleWritingMessage(self, msg):
             self.ExperimentStateIncrementer.resetStateLabels()
 
 
+class CountRateColorBar(QtWidgets.QWidget):
```

Comment:
I'm not going to object to this, but a colorbar seems unnecessary to me given that we show the actual count rate on each pixel.